README.md

```diff
@@ -1,107 +1,68 @@
-<div align="center">
-  <h1 align="center">
-    <img src="./www/static/img/logo-text.png" alt="tRPC" height="100" />
-  </h1>
-  <p>End-to-end typesafe APIs made easy</p>
-  <p>
-    <a href="https://codecov.io/gh/trpc/trpc">
-      <img src="https://codecov.io/gh/trpc/trpc/branch/main/graph/badge.svg?token=KPPS918B0G" alt="codecov">
-    </a>
-  </p>
-  <p>
-    <figure>
-      <img src="https://storage.googleapis.com/trpc/trpcgif.gif" alt="Server/client example" />
-      <figcaption>
-        The client above is <strong>not</strong> importing any code from the server, only it's type declarations.
-        <br/>
-        <sub><sup><em><a href="https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export">Import type only imports declarations to be used for type annotations and declarations. It always gets fully erased, so there’s no remnant of it at runtime.</a></em></sup></sub>
-      </figcaption>
-    </figure>
-  </p>
-</div>
-
-
-> 💬 Join the tRPC Discord server to chat to other people using tRPC - [trpc.io/discord](https://trpc.io/discord)
-
-## Intro
-
-
-tRPC allows you to easily build & consume fully typesafe APIs, without schemas or code generation.
-
-- 🧙‍♂️&nbsp; Full static typesafety & autocompletion on the client - on inputs, outputs, & errors.
-- 🐎&nbsp; Snappy DX. No code generation, run-time bloat, or build pipeline.
-- 🍃&nbsp; Light. tRPC has zero deps and a tiny client-side footprint.
-- 🐻&nbsp; Easy to add to your existing brownfield project.
-- 🔋&nbsp; Batteries included. React-library + Next.js/Express adapters. _(But tRPC is not tied to React - [reach out](https://twitter.com/alexdotjs) if you want to make a Svelte/Vue/... lib)_
-- 🥃&nbsp; Simple to use APIs for queries, mutations, & subscriptions support.
-- ⚡️&nbsp; Request batching - requests made at the same time can be automatically combined into one
-
-
-... and:
-
-- 👀&nbsp; Quite a few examples in the [./examples](./examples)-folder
-- ✅&nbsp; It's well-tested & running in production.
-
-
-> Still reading? Follow [**@alexdotjs** on Twitter](https://twitter.com/alexdotjs) if you have any questions or want to keep up to date what's coming next.
-
----
-
-- [Intro](#intro)
-- [Usage](#usage)
-- [Core Team](#core-team)
-- [Financial Contributors](#financial-contributors)
-  - [🥇 Gold Sponsors](#-gold-sponsors)
-  - [🥈 Silver Sponsors](#-silver-sponsors)
-  - [🥉 Bronze Sponsors](#-bronze-sponsors)
-  - [😻 Individuals](#-individuals)
-- [All contributors ✨](#all-contributors-)
```

Comment:
I wouldn't completely remove the TOC, but this can be shortened. I don't think most people care about the different sponsor levels, one link to that section should be fine.

Reply:
That's fair actually. I was thinking it'd be good if more sections are added to the README, but the README isn't long and those sections would probably be added on the docs website instead.